software/tubearchivist.yml

```diff
@@ -0,0 +1,19 @@
+name: "Tube Archivist"
+website_url: "https://tubearchivist.com/"
+source_code_url: "https://github.com/tubearchivist/tubearchivist"
+description: "Your self hosted YouTube media server"
```

Comment:
```suggestion
description: "By indexing your video collection with metadata from YouTube, you can organize, search and enjoy your archived YouTube videos without hassle offline through a convenient web interface."
```
Better description and ends with a dot

Reply:
This description doesn't seem appropriate to me. The set description is the exact description given by the project itself, it you just want a dot at the end I can do that.

Reply:
![image](https://github.com/awesome-selfhosted/awesome-selfhosted-data/assets/70863536/7e2a57ce-8bf4-4dff-9b9d-50897e45c1d7)
If it makes sense, then yeah. I was trying to make it avoid redundant words, etc.